homework-1/src/templates/slider.html

```diff
@@ -0,0 +1,90 @@
+<div class="slider">   
+   <div class="slider__area-logo">
+      <a id="sliderLogo" class="slider__logo-link"><img src="/assets/images/SkoutGlossary.png" alt="" class="slider__logo" ></a>
+   </div>
+   <div class="slider__section">
+            <div class="slider__left">
+                <div class="slider__left-one" id="sliderLeftOne">
+                   <div class="slider__dots">
```

Comment:
dots - bad class name. This list with text and lines have nothing common with dots